securedrop/requirements/update_python_dependencies

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./update-python-dependencies
```

Comment:
@fowlslegs I'm looking into https://docs.securedrop.org/en/latest being down. FWIW, I notice that https://securedrop.readthedocs.io/en/latest/ is also down, so I think the problem is with Read the Docs, not our DNS.